scripts/deployments/RunWorkflows.ps1

```diff
@@ -75,7 +75,7 @@ OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
   .PARAMETER GitHubRef
     The GitHub ref to use for the workflow.
 
-  .PARAMETER LoginInteractiveTenantId
+  .PARAMETER LoginInteractiveTenentId
```

Comment:
The existing spelling is correct.  See https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-to-find-tenant